AdyenActions/Components/SDK/TwintSDKActionComponent.swift

```diff
@@ -121,10 +122,15 @@ import Foundation
                 message: error,
                 preferredStyle: .alert
             )
-            alert.addAction(.init(
-                title: localizedString(.dismissButton, configuration.localizationParameters),
-                style: .default
-            )
+            alert.addAction(
+                .init(
+                    title: localizedString(.dismissButton, configuration.localizationParameters),
+                    style: .default, 
+                    handler: { _ in
+                        // TODO: Is this the right thing to do here? - we have to let drop-in know to hide the spinner without dismissing drop-in
```

Comment:
Any opinion on this part here?
I believe this is the right thing to do to stop drop-in from loading

Reply:
you mean to hide the spinner next to  twint PM?

Reply:
Yes exactly. The dismissing of the alert should just hide the spinner but not dismiss drop-in I assume.

Reply:
yes we should do this :)

Reply:
Ok I will get rid of the discussion todo 👌